<?xml version="1.0"?>
<!--
  Universal robot common bringup.  Starts ur driver node and robot state
  publisher (translates joint positions to propper tfs).

  Usage:
    ur_common.launch robot_ip:=<value>
-->
<launch>
  <!-- robot_ip: IP-address of the robot's socket-messaging server -->
  <arg name="robot_ip" />
  <arg name="min_payload" />
  <arg name="max_payload" />
  <arg name="prefix" default="" />
  <arg name="servoj_time" default="0.008" />
  <arg name="base_frame" default="$(arg prefix)base" />
  <arg name="tool_frame" default="$(arg prefix)tool0_controller" />

  <!-- The max_velocity parameter is only used for debugging in the ur_driver. It's not related to actual velocity limits -->
  <arg name="max_velocity" default="10.0"/> <!-- [rad/s] -->

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />

</launch>
